.env.sample

```diff
@@ -76,9 +77,21 @@ DJANGO_LOGGING_LEVEL=INFO
 # CACHE_LOCATION=redis://student_explorer_redis:6379/1
 
 # ** Database CACHE_LOCATION is a table name in the 'default' database (MySQL)
+# This table is created automatically and could be anything.
 # CACHE_BACKEND=django.core.cache.backends.db.DatabaseCache
 # CACHE_LOCATION=se_view_cache
 
+# ** MySQL CACHE_LOCATION is created by a migration and has to be this table
+# This library supports CACHE_OPTIONS, see this for more info:
+# https://django-mysql.readthedocs.io/en/latest/cache.html
+# CACHE_BACKEND=django_mysql.cache.MySQLCache
+# CACHE_LOCATION=django_mysql_cache
```

Comment:
I am lurking here, what is the purpose of 2 cache `redis` and `mysql`? are bother serving different purposes that one doesn't have

Reply:
Just depends in which one we end up using.

We already have a MySQL server so we can just cache to a key/value store table in the MySQL database. This may be enough. I'm leaving Redis support in here (just the package) incase we need to stand up Redis instead because MySQL isn't working well enough (need more speed).

It's probably more of a tradeoff of performance for cost. If we just end up staying on MySQL we can remove the Redis. Django just gives you the options to use whatever you want, but you have to have the packages available to support them.